bin/storm-config.cmd

```diff
@@ -83,10 +83,10 @@ if not defined STORM_LOG_DIR (
 )
 
 @rem
-@rem retrieve storm.logback.conf.dir from conf file
+@rem retrieve storm.log4j2.conf.dir from conf file
 @rem
 
-"%JAVA%" -client -Dstorm.options= -Dstorm.conf.file= -cp "%CLASSPATH%" backtype.storm.command.config_value storm.logback.conf.dir > %CMD_TEMP_FILE%
+"%JAVA%" -client -Dstorm.options= -Dstorm.conf.file= -cp "%CLASSPATH%" backtype.storm.command.config_value storm.log4j2.conf.dir > %CMD_TEMP_FILE%
   
```

Comment:
I was looking around and found other usages of LOCKBACK in the storm-config.cmd file.  May as well change those while we are changing the others.